python/ray/tune/integration/wandb.py

```diff
@@ -355,7 +354,8 @@ def _init(self):
                 "You need to specify a `project` in your wandb `config` dict.")
 
         # Grouping
-        wandb_group = wandb_config.pop("group", self.trial.trainable_name)
+        wandb_group = wandb_config.pop(
+            "group", self.trial.trainable_name if self.trial else None)
```

Comment:
btw I think we should be grouping by experiment name but this works for now (since it is not a regression).